src/tests/sorting.spec.js

```diff
@@ -0,0 +1,48 @@
+import { test, expect } from '../fixtures/base';
+
+test.describe('sorting invertory page', () => {
+    test('sorting name A-Z', async ({ page, app }) => {
+        await app.login.navigate('/');
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        await expect(page).toHaveURL(app.inventory.url);
+
+        await app.inventory.sortBy('az');
+
+        await expect(app.inventory.inventoryItems.nth(0).getByTestId('inventory-item-name')).toHaveText('Sauce Labs Backpack');
+    });
+
+    test('sorting name Z-A', async ({ page, app }) => {
+        await app.login.navigate('/');
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        await expect(page).toHaveURL(app.inventory.url);
+
+        await app.inventory.sortBy('za');
```

Comment:
Please, avoid using magic strings.
Only you know that the name of 'za' or 'lohi'
It is better to have some matcher to use it like 
`sortBy(sortingOptions.nameDesc)`
or 
`sortBy(sortingOptions.priceAsc)`